test/unit/knex.js

```diff
@@ -186,6 +187,10 @@ describe('knex', () => {
   });
 
   it('passes queryContext to wrapIdentifier in raw query', () => {
+    if (!sqliteConfig) {
```

Comment:
why could it be not available?

Reply:
when someone passes for example `DB=oracledb` environment variable. In that case sqliteConfig is undefined.